src/main/java/org/templegalaxia/patterns/acjs/Spoken.java

```diff
@@ -0,0 +1,216 @@
+package org.templegalaxia.patterns.acjs;
+
+import heronarts.lx.LX;
+import heronarts.lx.LXCategory;
+import heronarts.lx.LXLayer;
+import heronarts.lx.color.LXColor;
+import heronarts.lx.modulator.LXPeriodicModulator;
+import heronarts.lx.modulator.SinLFO;
+import heronarts.lx.parameter.CompoundParameter;
+import heronarts.lx.parameter.LXParameter;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Stream;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.templegalaxia.patterns.TemplePattern;
+
+/*
+ * “Do you not know that a man is not dead while his name is still spoken?”
+ *         -- Terry Pratchett, Going Postal
+ */
+
+@LXCategory("Acjs")
+public class Spoken extends TemplePattern {
+  private static final String NAME_DATA = "/Users/anton/Projects/Galaxia/Galaxia/assets/names.txt";
```

Comment:
Don't think you should use an absolute path here... Processing has helper methods that load files from the base directory, refer to this as "assets/names.txt" - you might also want to use a filename here that is more obviously related to the pattern file, e.g. Spoken.txt